docs/components/attention/react.md

```diff
@@ -98,6 +113,32 @@ function Example() {
 }
 ```
 
+### Accessibility
+
+The attention element handles accessibility automatically by wrapping its slotted content with a `div` that has a default `role`attribute (`role="tooltip"` for tooltip and `role="img"` otherwise), and a default localized `aria-label`.
+
+It is possible to reset the `role` and `aria-label` attributes, and the default `aria-label` can be overridden:
+
+```js
+<div className="flex items-center">
+  <Box info>
+    <h1 aria-details='callout-bubbletext'>I am a box full of info</h1>
+  </Box>
+  <Attention
+    callout
+    placement='right'
+    isShowing={true}
+    role=''
+    aria-label='overriding default aria-label'
+  >
+  <p id='callout-bubbletext' role='img' style={{ width: 200 }}>
+    I'm a callout speech bubble with resetted role and overridden aria-label
+    attributes pointing left.
+  </p>
+  </Attention>
```

Comment:
According to MDN, `role="img"` should be used to "identify multiple elements inside page content that should be considered as a single image. These elements could be images, code snippets, text, emojis, or other content that can be combined to deliver information in a visual manner."
Based on that and on the previous a11y examples from the Elements docs, I would maybe change the Attention type to `tooltip`, set `aria-label=""` on the component, and set `role="tooltip"` on the `<p>` tag. The ids would need to be updated to `tooltip-bubbletext` for the sake of consistency :)

Reply:
Wait do you mean that I should change the example to show a tooltip-example instead of a callout-example?